### PR TITLE
Flatten AVIRIS data catalog

### PR DESCRIPTION
**Pull request by CloudNiner**
## Overview

Having nested collections ingested into Franklin doesn't really gain us anything, since we can use the query extension to filter on the Year and Flight properties for the same effect. It also cluttered the collections endpoint. This PR flattens the AVIRIS catalog to a single root collection. I've already ingested it into the NASA Franklin instance.

Connects #55

## Demo

Screen Shot 2020-12-07 at 4 07 00 PM

## Testing Instructions

Generate AVIRIS data catalog. It should be a flat catalog with a Collection as root named `aviris-data`.

## Checklist

- [x] `fixup!` commits have been squashed
- [x] CI passes after rebase
- [x] README.md updated if necessary to reflect the changes

### CODE REVIEW

File: catalogs/README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ This folder contains a series of scripts used to generate STAC Catalogs for this
 
 | Catalog Name | Folder | Published Location |
 |--------------|--------|--------------------|
-| AVIRIS       | aviris | `s3://aviris-data/stac-catalog/catalog.json` |
+| AVIRIS       | aviris | `s3://aviris-data/stac-catalog/collection.json` |
```
Comment (pomadchin) [inline catalogs/README.md:9]: @CloudNiner can we still add a single `catalog.json` file that would have a reference to a single collection?

**Review by pomadchin — COMMENTED** [review]
LGTM and solves the issue! Franklin catalog looks as expected. However, I'm not sure that it generates a valid static catalog. Is a STAC Catalog without a `Catalog` a valid one?

File: catalogs/README.md (hunk 1 above)
Comment (CloudNiner) [reply catalogs/README.md]: This generated some internal discussion that led to https://github.com/radiantearth/stac-spec/issues/925

For now I've reverted to a single Collection within a Catalog at the root for consistency with other catalogs such as https://landsat-stac.s3.amazonaws.com/catalog.json

@pomadchin if this looks good I'll merge and begin the reupload and reingest into Franklin

**Review by pomadchin — APPROVED** [review]
LGTM, thanks!

**Pull request merged**